test/merchant_test.rb

```diff
@@ -1,5 +1,7 @@
 require_relative 'test_helper'
 require './lib/merchant'
+require './lib/sales_engine'
+require './lib/merchant_repository'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.